plugins/woocommerce-admin/client/products/shared/add-new-shipping-class-modal/add-new-shipping-class-modal.tsx

```diff
@@ -39,13 +39,9 @@ function ShippingClassForm( { onAdd, onCancel }: ShippingClassFormProps ) {
 		<div className="woocommerce-add-new-shipping-class-modal__wrapper">
 			<TextControl
 				{ ...getInputProps( 'name' ) }
-				label={ __( 'Name', 'woocommerce' ) }
-			/>
-			<TextControl
-				{ ...getInputProps( 'slug' ) }
 				label={ interpolateComponents( {
 					mixedString: __(
-						'Slug {{span}}(optional){{/span}}',
+						'Name {{span}}(required){{/span}}',
```

Comment:
I wonder if passing just an `{{required/}}` component would make this slightly easier on translators.

E.g.,
```
components: {
  optional: {
    <span className="woocommerce-add-new-shipping-class-modal__optional-input">{ __( '(required)', 'woocommerce' ) }</span>
  }
}
```

Reply:
Cool!!! This is way better. Thanks and Done!!!